mappings/net/minecraft/client/network/ClientLoginNetworkHandler.mapping

```diff
@@ -25,5 +25,5 @@ CLASS net/minecraft/class_635 net/minecraft/client/network/ClientLoginNetworkHan
 		ARG 2 packet
 		ARG 3 decryptionCipher
 		ARG 4 encryptionCipher
-	METHOD method_51768 setRealmsMinigameName (Ljava/lang/String;)V
-		ARG 1 realmsMinigameName
+	METHOD method_51768 setMinigameName (Ljava/lang/String;)V
```

Comment:
Do you think we need the `Realms` prefix to make it clear? I personally don't know what a minigame name is in Minecraft unless realms is specified.